apax/model/gmnn.py

```diff
@@ -223,13 +242,32 @@ def __call__(
         }
 
         if self.force_variance:
-            forces_ens = -jax.jacrev(self.energy_model)(R, Z, neighbor, box, offsets)
+            if not self.chunk_size:
+                forces_ens = -jax.jacrev(self.energy_model)(R, Z, neighbor, box, offsets)
+            else:
+                with jax.ensure_compile_time_eval():
+                    assert n_ens % self.chunk_size == 0
+
```

Comment:
assertion should not be used, can this be a value or runtime error?

Reply:
yes